### PR TITLE
Data and scripts to make fig5, fig6, figS4d, figS6, figS7, tableS2, a…

### PR DESCRIPTION
**Pull request by niclak**
…nd tableS3

### CODE REVIEW

File: manuscript_analyses/collate_Figure5.R [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#compile figure panel
```
Comment (klaricch) [inline manuscript_analyses/collate_Figure5.R:1]: for all R scripts, doesn't need to be changed, but typically would add spaces around operators and after commas

**Review by klaricch — CHANGES_REQUESTED** [review]

File: manuscript_analyses/final_data_files/insilicos/get_hmtvar.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        for row in vcf:
+            pos = row["POS"]
+            ref = row["REF"]
+            alt = row["ALT"]
+            variant = ref + pos + alt
+
+            url = "https://www.hmtvar.uniba.it/api/main/mutation/" + variant
+
+            output = urllib.urlopen(url)
+            annotation = output.read()
```
[the remark this answers is not in the document]
Comment (niclak) [reply manuscript_analyses/final_data_files/insilicos/get_hmtvar.py:30]: removed the new line in the write below to do this

File: manuscript_analyses/reformat_vcf.py [hunk 3]
```diff
@@ -0,0 +1,793 @@
...
+                    + str(mitomap_hom)
+                    + "\n"
+                )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "-other_databases_path",
```
[the remark this answers is not in the document]
Comment (niclak) [reply manuscript_analyses/reformat_vcf.py:722]: changed all to --

File: manuscript_analyses/reformat_vcf.py [hunk 4]
```diff
@@ -0,0 +1,793 @@
...
+    with open(insilicos_path + "hmtvar_annotations.txt") as hmtvar:
+        hmtvar = csv.DictReader(hmtvar, delimiter="\t")
+
+        matches_hmtvar = {}
+
+        for row in hmtvar:
+            annotation = row["HmtVar"]
+            insilico = ""
+            # extract the in silico prediction from the annotation
+            if len(annotation) > 3:
```
[the remark this answers is not in the document]
Comment (niclak) [reply manuscript_analyses/reformat_vcf.py:83]: thanks for the pointer - I've updated the code

File: manuscript_analyses/reformat_vcf.py [hunk 5]
```diff
@@ -0,0 +1,793 @@
...
+                            ]  # haplogroup of sample
+                            haplos.append(haplogroup)
+                            heteroplasmy = cell.split(":")[2]
+                            all_het.append(heteroplasmy)
+
+            if row[6] == "PASS":
+                POS = row[1]
+                REF = row[3]
+                ALT = row[4]
+                info = row[7]  # column 8
```
[the remark this answers is not in the document]
Comment (niclak) [reply manuscript_analyses/reformat_vcf.py:157]: good idea - i've updated the code to do this

File: manuscript_analyses/reformat_vcf.py [hunk 6]
```diff
@@ -0,0 +1,793 @@
...
+                        hmtvar = ""
+
+                    file.write(
+                        str(POS)
+                        + "\t"
+                        + str(REF)
+                        + "\t"
+                        + str(ALT)
+                        + "\t"
+                        + str(matches_annotations[POS, REF, ALT, SYMBOL][0])
```
[the remark this answers is not in the document]
Comment (niclak) [reply manuscript_analyses/reformat_vcf.py:378]: have updated to do this

**Review by klaricch — COMMENTED** [review]
just a few small things

**Pull request merged**